kbase/narrative
# Dataup 224 checkbox tooltip

![Screen Shot 2020-11-09 at 12 44 24 PM](https://user-images.githubusercontent.com/4184019/98594314-56f9af80-2289-11eb-9bd7-30856db941d6.png)

# Description of PR purpose/changes

Adding tooltips to disabled checkboxes that say "Select a type to continue" This was a requirement that got missed on the original ticket. 

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-224

# Testing Instructions
* Details for how to test the PR: 
- Open up the data import pane on any narrative
- Add new files (unless you already have some without a type selected)
- Those which do not have a type selected should have the left hand checkbox disabled, when you hover over it you should see a tooltip that says "Select a type to continue."

# Dev Checklist:
